src/Form/index.tsx

```diff
@@ -608,12 +608,22 @@ function Form({
       }
     }
     newStep = JSON.parse(JSON.stringify(newStep));
-    if (
-      contextRef &&
-      Object.prototype.hasOwnProperty.call(contextRef, 'current')
-    )
-      // Need to update the context each time because it has the step closure'd in
-      contextRef.current = getFormContextHelper(newStep);
+
+    initState.formState[formUuid] = {
```

Comment:
make a new internalState global for these

Reply:
add comment in internalState and here about what these properties are for
explain need for context & having up to date values during runtime